### PR TITLE
Create Admin Account

### PR DESCRIPTION
**Pull request by Nasser-Eddin-Nasser**

### CODE REVIEW

File: HR-Monitor/hr-monitor.properties [hunk 1]
```diff
@@ -0,0 +1,4 @@
+path.variable.kotlin_bundled=C\:\\Program Files\\JetBrains\\IntelliJ IDEA Community Edition 2019.2.3\\plugins\\Kotlin\\kotlinc
+path.variable.maven_repository=C\:\\Users\\EDDIN\\.m2\\repository
+jdk.home.11.0.4=C\:/Program Files/JetBrains/IntelliJ IDEA Community Edition 2019.2.3/jbr
+javac2.instrumentation.includeJavaRuntime=false
```
Comment (vishwasanavatti) [inline HR-Monitor/hr-monitor.properties:4]: what is this file for? why are paths hard coded?

**Review by vishwasanavatti — CHANGES_REQUESTED** [review]
Tested locally. I could able to create an account but was not able to login. Could you check the issue?

File: HR-Monitor/hr-monitor.properties (hunk 1 above)
Comment (Nasser-Eddin-Nasser) [reply HR-Monitor/hr-monitor.properties:4]: It's for the Ant builder, it should be in git ignore I guess :)

**Review by vishwasanavatti — APPROVED** [review]
Tested again after changes and its works fine. Good work @Nasser-Eddin-Nasser, its perfectly done. All the basic functionalities are included. Can be merged.

**Review by dalex135 — APPROVED** [review]
Ran and tested. Well done. Quality code. :)
Was easy to merge with the US 42.

**Comment by MBuchalik** [comment]
Edit: Sorry - I missed someting. I initially thought the passwords would be stored in cleartext :D (Or that some sort of encryption would be used instead of a hash.)

Would be cool if we used sth like a salted hash, but we can add this in the future as well.

**Pull request merged**